snaps/openvino/consumer-intel-npu-driver-and-openvino-toolkit-2404/snap/snapcraft.yaml

```diff
@@ -16,7 +16,7 @@ plugs:
   npu-libs:
     interface: content
     content: npu-libs-2404
-    target: $SNAP/usr/lib/npu-libs-2404
+    target: $SNAP/npu-libs
```

Comment:
interested by the thought motivating this renamig

Reply:
Just to simplify the path as much as possible and for consistency with the openvino content snap, which uses `$SNAP/openvino`.